bitnami/redis/templates/health-configmap.yaml

```
@@ -12,35 +12,49 @@ metadata:
   {{- end }}
 data:
   ping_readiness_local.sh: |-
-    #!/bin/bash
+    #!/usr/bin/env bash
```

In this base image, Bash is guaranteed to be located at:
```suggestion
    #!/bin/bash
```